bitbot/dice.go

```diff
@@ -0,0 +1,40 @@
+package bitbot
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/justinian/dice"
+	"github.com/whyrusleeping/hellabot"
+	log "gopkg.in/inconshreveable/log15.v2"
+)
+
+const DICE_USAGE = "Usage: [num dice]d[sides](+/-num) (opt: if fudging)"
+
+var RollTrigger = hbot.Trigger{
```

Comment:
Use a named trigger instead of `hbot.Trigger`
```suggestion
var RollTrigger = hbot.Trigger{
  ID: "roll"
```